central/complianceoperator/v2/scanconfigurations/service/service_impl.go

```diff
@@ -0,0 +1,136 @@
+	"github.com/stackrox/rox/central/cluster/datastore"
+	"github.com/stackrox/rox/central/complianceoperator/v2/compliancemanager"
+	complianceDS "github.com/stackrox/rox/central/complianceoperator/v2/scanconfigurations/datastore"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/grpc/authz"
+	"github.com/stackrox/rox/pkg/grpc/authz/perrpc"
+	"github.com/stackrox/rox/pkg/grpc/authz/user"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/paginated"
+	"google.golang.org/grpc"
+)
+
+const (
+	maxPaginationLimit = 1000
+)
+
+var (
+	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
+		user.With(permissions.View(resources.Compliance)): {
+			"/v2.ComplianceScanConfigurationService/ListComplianceScanConfigurations",
+			"/v2.ComplianceScanConfigurationService/GetComplianceScanConfiguration",
+		},
+		user.With(permissions.Modify(resources.Compliance)): {
+			"/v2.ComplianceScanConfigurationService/PostComplianceScanConfiguration",
+			"/v2.ComplianceScanConfigurationService/DeleteComplianceScanConfiguration",
+			"/v2.ComplianceScanConfigurationService/RunComplianceScanConfiguration",
+			"/v2.ComplianceScanConfigurationService/UpdateComplianceScanConfiguration",
+		},
+	})
+	log = logging.LoggerForModule()
+)
+
+// New returns a service object for registering with grpc.
+func New(complianceScanSettingsDS complianceDS.DataStore, manager compliancemanager.Manager, clusterStore datastore.DataStore) Service {
+	return &serviceImpl{
+		complianceScanSettingsDS: complianceScanSettingsDS,
+		manager:                  manager,
+		clusterStore:             clusterStore,
+	}
+}
+
+type serviceImpl struct {
+	v2.UnimplementedComplianceScanConfigurationServiceServer
+
+	complianceScanSettingsDS complianceDS.DataStore
+	manager                  compliancemanager.Manager
+	clusterStore             datastore.DataStore
+}
+
+// RegisterServiceServer registers this service with the given gRPC Server.
+func (s *serviceImpl) RegisterServiceServer(grpcServer *grpc.Server) {
+	v2.RegisterComplianceScanConfigurationServiceServer(grpcServer, s)
+}
+
+// RegisterServiceHandler registers this service with the given gRPC Gateway endpoint.
+func (s *serviceImpl) RegisterServiceHandler(ctx context.Context, mux *runtime.ServeMux, conn *grpc.ClientConn) error {
+	return v2.RegisterComplianceScanConfigurationServiceHandler(ctx, mux, conn)
+}
+
+// AuthFuncOverride specifies the auth criteria for this API.
+func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
+	return ctx, authorizer.Authorized(ctx, fullMethodName)
+}
+
+func (s *serviceImpl) PostComplianceScanConfiguration(ctx context.Context, req *v2.ComplianceScanConfiguration) (*v2.ComplianceScanConfiguration, error) {
```

Comment:
`Post...` -> `Create...`

For more details: https://github.com/stackrox/stackrox/blob/master/.github/api_guidelines.md